scripts/sysman/virtualFunctionManagement.yml

```diff
@@ -0,0 +1,252 @@
+    - type: $s_vf_info_util_flag_exp_t
+      name: "flag"
+      desc: "[out] Bit field of utilization flags available."
+--- #--------------------------------------------------------------------------
+type: struct
+desc: "Provides memory utilization values for a virtual function"
+version: "1.9"
+class: $sVFManagement
+name: $s_vf_util_mem_exp_t
+base: $s_base_state_t
+members:
+    - type: $s_vf_info_mem_type_exp_t
+      name: "memtype"
+      desc: "[out] Memory type flags."
+    - type: uint64_t
+      name: "free"
+      desc: "[out] Free memory size in bytes."
+    - type: uint64_t
+      name: "size"
+      desc: "[out] Total allocatable memory in bytes."
+    - type: uint64_t
+      name: "timestamp"
+      desc: "[out] Wall clock time from VF when value was sampled."
+--- #--------------------------------------------------------------------------
+type: struct
+desc: "Provides engine utilization values for a virtual function"
+version: "1.9"
+class: $sVFManagement
+name: $s_vf_util_engine_exp_t
+base: $s_base_state_t
+members:
+    - type: $s_engine_group_t
+      name: "type"
+      desc: "[out] The engine group"
+    - type: uint64_t
+      name: "activeCounterValue"
+      desc: "[out] Represents active counter."
+    - type: uint64_t
+      name: "samplingCounterValue"
+      desc: "[out] Represents counter value when activeCounterValue was sampled."
+    - type: uint64_t
+      name: "timestamp"
+      desc: "[out] Wall clock time when the activeCounterValue was sampled."
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Get handle of virtual function modules"
+class: $sDevice
+name: EnumActiveVirtualFunctionExp
+details:
+    - "The application may call this function from simultaneous threads."
+    - "The implementation of this function should be lock-free."
+params:
+    - type: $s_device_handle_t
+      name: hDevice
+      desc: "[in] Sysman handle of the device."
+    - type: "uint32_t*"
+      name: pCount
+      desc: |
+            [in,out] pointer to the number of components of this type.
+            if count is zero, then the driver shall update the value with the total number of components of this type that are available.
+            if count is greater than the number of components of this type that are available, then the driver shall update the value with the correct number of components.
+    - type: "$s_vf_handle_t*"
+      name: pVFhandle
+      desc: |
+            [in,out][optional][range(0, *pCount)] array of handle of components of this type.
+            if count is less than the number of components of this type that are available, then the driver shall only retrieve that number of component handles.
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Get virtual function management properties"
+class: $sVFManagement
+name: GetVFPropertiesExp
```

Comment:
Should this be GetVf instead of GetVF ?

Reply:
Since it is only 2 letters that reference 2 commonly understood words, I think it is ok to leave both caps.